src/Microsoft.Diagnostics.Monitoring.Egress.AzureStorage/AzureBlobEgressProviderOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Monitoring.Egress.AzureStorage
+{
+    internal class AzureBlobEgressProviderOptions :
+        EgressProviderOptions
+    {
+        public string ContainerName { get; set; }
+
+        public string AccountUri { get; set; }
```

Comment:
Can you type this as a `Uri` instead of a string?

Reply:
Sure can, just verified that the configuration binding will allow it.

Reply:
Done